posts/2017-05-10-serverless-v1.13.0.md

```diff
@@ -0,0 +1,157 @@
+---
+title: New in Serverless v1.13 - v1.13 opens up world of configuration possibilities with new async variable support
```

Comment:
Title too long and repeats =)

How about:

"Serverless v1.13 - Async variable support added for easier configuration & secret management"